### PR TITLE
Hi-fi: Contact

### PR DESCRIPTION
**Pull request by jzhzhang66**
*Note*: remove header and footer from contact.js for most accurate results since those will be styled later 
**Confirmation popup is being pulled into a separate PR for modifications so please disregard that for now!**
contact.js: 
     - removed all rendering code (only grabs data and passes to navigation,  contact page, and footer components
contactPage.js: 
     - component to render all the contents on the contact page
contactConfirmation.js: 
     - component to handle the confirmation pop up when submit is clicked 
contactForm.js: 
     - component for the contact form (contains all input fields on contact page and submit button) 
textInputBox.js
     - component to render an input box 
boxLink.js:  
     - component to render a box link with title, brief description, and a link to guide to another page 
backToStartVerticalButton.js: 
     - button to redirect user to the top (clicking animation will be added when we start working on animations)

### CODE REVIEW

File: components/boxLink/BoxLink.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import MediaQuery from 'react-responsive';
+export default function BoxLink({ boxLinkData }) {
+  const {
+    box_link_title: boxLinkTitle,
+    box_link_description: boxLinkDescription,
+    box_link_link: boxLinkLink,
+  } = boxLinkData;
+
+  return (
+    <>
```
Comment (isabellberzsenyi) [inline components/boxLink/BoxLink.js]: don't think empty tag is necessary here

File: components/boxLink/BoxLink.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  Description,
+  MediumArrow,
+  LongArrow,
+  InnerContentContainer,
+} from './BoxLink.styles';
+import { icons } from '../../style/icons';
+import PageLink from '../pageLink/PageLink';
+import { breakpointsObj } from '../../lib/responsive';
+import MediaQuery from 'react-responsive';
+export default function BoxLink({ boxLinkData }) {
```
Comment (isabellberzsenyi) [inline components/boxLink/BoxLink.js:16]: formatting: can you add an empty line between imports and `export default function BoxLink...`

File: components/backToStartVerticalButton/BackToStartVerticalButton.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  Button,
+  ButtonText,
+  UnfilledButtonImage,
+  FilledButtonImage,
+} from './BackToStartVerticalButton.styles';
+import { icons } from '../../style/icons';
+import React, { useState } from 'react';
+import { breakpointsObj } from '../../lib/responsive';
+import MediaQuery from 'react-responsive';
+export default function BackToStartVerticalButton({ onClick }) {
```
Comment (isabellberzsenyi) [inline components/backToStartVerticalButton/BackToStartVerticalButton.js:12]: formatting: please add empty line after imports (please do for all components, this helps w visibility

File: components/squareButton/SquareButton.styles.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import { P } from '../../style/typography';
+import styled from 'styled-components';
+import { colors } from '../../style/colors';
+import { fontWeights } from '../../style/typography';
+import { min } from '../../lib/responsive';
+
+const Button = styled.button`
+  background-color: ${colors.OLIVE};
+  color: white;
```
Comment (isabellberzsenyi) [inline components/squareButton/SquareButton.styles.js]: let's use the white from colors: `${colors.WHITE}`

File: components/contactPage/contactForm/ContactForm.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+import SquareButton from '../../squareButton/SquareButton';
+import {
+  ContactFormContainer,
+  InputFieldWrapper,
+  InputBoxWrapper,
+  ButtonWrapper,
+} from './ContactForm.styles';
+export default function ContactForm({ togglePopup }) {
+  return (
+    <>
```
Comment (isabellberzsenyi) [inline components/contactPage/contactForm/ContactForm.js:12]: empty tag not necessary here

File: components/textInputBox/TextInputBox.styles.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  background-color: ${colors.CREME};
+  width: 100%;
+  height: clamp(16em, 4vw, 26em);
+  border: 1px solid ${colors.BROWN};
+  &:focus {
+    outline: none;
+  }
+  color: ${colors.BROWN};
+  font-family: ${fonts.poppins};
+  font-weight: ${fontWeights.regular};
```
Comment (isabellberzsenyi) [inline components/textInputBox/TextInputBox.styles.js]: should be `fontWeights.normal`

File: components/contactPage/contactConfirmationPopup/ContactConfirmationPopup.styles.js [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+  @media ${min.desktopLg} {
+    top: 0;
+  }
+`;
+
+const Box = styled.div`
+  background: ${colors.CREME};
+  position: relative;
+  margin: 0 auto;
+  border: 0.15em solid black;
```
Comment (isabellberzsenyi) [inline components/contactPage/contactConfirmationPopup/ContactConfirmationPopup.styles.js]: let's use `${colors.BLACK}` instead

**Comment by isabellberzsenyi** [comment]
Looks great! Just a couple minor comments above, and some over all modifications I recommend:

- empty tags are not necessary if it is wrapping just one element (ex. my comment for `BoxLink.js`)
- please add empty lines after the imports, helps with readability
- I have some hesitation with using media queries for `min.mobile` because it leaves the minor edge case if a screen is less than `min.mobile`. I can explain more on a call & would love to hear others' thoughts

**Review by isabellberzsenyi — CHANGES_REQUESTED** [review]
Looks great! Just a couple minor comments above, and some over all modifications I recommend:

- empty tags are not necessary if it is wrapping just one element (ex. my comment for BoxLink.js)
- please add empty lines after the imports, helps with readability
- I have some hesitation with using media queries for min.mobile because it leaves the minor edge case if a screen is less than min.mobile. I can explain more on a call & would love to hear others' thoughts

File: components/backToStartVerticalButton/BackToStartVerticalButton.styles.js [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+  @media ${min.tablet} {
+    width: 70%;
+  }
+  @media ${min.desktop} {
+    width: 42%;
+  }
+`;
+
+const Button = styled.button`
+  background-color: Transparent;
```
Comment (josephmannis) [inline components/backToStartVerticalButton/BackToStartVerticalButton.styles.js]: Nitpick but can you make this lowercase

File: components/boxLink/BoxLink.js (hunk 1 above)
Comment (josephmannis) [reply components/boxLink/BoxLink.js]: I think you mean don't think empty tag is necessary hehe

File: components/contactPage/ContactPage.js [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+          italicized={contactPageTitleItalic}
+        ></H1Styled>
+        <SquiggleUnderline
+          src={icons.SQUIGGLE_UNDERLINE}
+          alt="Contact page header squiggle underline"
+        />
+      </HeaderAndSquiggleContainer>
+      <PageDescription
+        description={contactPageDescription}
+        arrowText="LET'S CONNECT"
```
Comment (josephmannis) [inline components/contactPage/ContactPage.js]: Could you verify that this shouldn't come from the CMS? This is the type of thing people do like to change

File: components/contactPage/contactConfirmationPopup/ContactConfirmationPopup.js [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+            <H2Styled>
+              {getString(contactConfirmationTitleRegular1) + ' '}
+              <i>{getString(contactConfirmationTitleItalic2) + ' '}</i>
+              {getString(contactConfirmationTitleRegular3)}
+            </H2Styled>
+            <PStyled>{getString(description)}</PStyled>
+          </TextContent>
+          <ImgBackground src={image.url} alt={image.alt} />
+          <Img src={image.url} alt={image.alt} />
+          <X
```
Comment (josephmannis) [inline components/contactPage/contactConfirmationPopup/ContactConfirmationPopup.js]: Could you name this something more descriptive please? CloseIcon perhaps

File: components/contactPage/contactForm/ContactForm.js [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+      <ContactFormContainer>
+        <InputFieldWrapper>
+          <TextInputField id="1" label="First Name" />
```
Comment (josephmannis) [inline components/contactPage/contactForm/ContactForm.js]: Could you please just make sure these have more human readable names/ids? The point of them is to let those with screen readers know what they're for. 

https://www.w3.org/WAI/tutorials/forms/

File: components/contactPage/contactForm/ContactForm.js [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+          <TextInputField id="2" label="Last Name" />
+        </InputFieldWrapper>
+        <InputFieldWrapper>
+          <TextInputField id="3" label="Email" />
+        </InputFieldWrapper>
+        <InputBoxWrapper>
+          <TextInputBox placeholder="Message" />
+        </InputBoxWrapper>
+        <ButtonWrapper>
+          <SquareButton buttonText="SUBMIT" onClick={togglePopup} />
```
Comment (josephmannis) [inline components/contactPage/contactForm/ContactForm.js]: this should have either an aria label saying it's the submission button or use an input with type submit

**Review by josephmannis — CHANGES_REQUESTED** [review]
A few little accessibility things

File: components/confirmationPopup/ConfirmationPopup.styles.js [hunk 13]
```diff
@@ -0,0 +1,129 @@
+import styled from 'styled-components';
```
Comment (jean-zhang) [inline components/confirmationPopup/ConfirmationPopup.styles.js:1]: I believe there is an overlay on the entire screen and the scrolling is stopped when the pop-up comes up, I did something similar on my PR for the Home Page if you want to take a look

File: components/textInputField/TextInputField.style.js [hunk 14]
```diff
@@ -11,10 +11,11 @@ const TextLabel = styled(P)`
 const TextInput = styled(P).attrs({
   as: 'input',
 })`
+  background-color: ${colors.CREME};
   display: block;
```
Comment (jean-zhang) [inline components/textInputField/TextInputField.style.js:15]: Text should be brown here as well, right now its black

**Review by jean-zhang — APPROVED** [review]

File: components/textInputBox/TextInputBox.styles.js [hunk 15]
```diff
@@ -0,0 +1,23 @@
...
+import { fonts, fontWeights } from '../../style/typography';
+const InputBox = styled.textarea`
+  background-color: ${colors.CREME};
+  width: 100%;
+  height: clamp(16em, 4vw, 26em);
+  border: 1px solid ${colors.BROWN};
+  &:focus {
+    outline: none;
+  }
+  color: ${colors.BROWN};
```
Comment (jean-zhang) [inline components/textInputBox/TextInputBox.styles.js]: Seems like HiFi has the text as black

**Review by isabellberzsenyi — APPROVED** [review]
great

**Pull request merged**